plugins/generator-1.15.2/forge-1.15.2/procedures/entity_get_shootpower.java.ftl

```diff
@@ -0,0 +1 @@
+(((projectileentity.getEntity() instanceof ${generator.map(field$projectiles, "projectiles")}) && (projectileentity != null))?((${generator.map(field$projectiles, "projectiles")}) projectileentity).getMotion().distanceTo(Vec3d.ZERO):0)
```

Comment:
You’re using the projectileentity variable before checking if it’s not null

Reply:
It's done. I changed the first check with `&&` in two checks with the `!= null` check first.